worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -435,15 +550,24 @@ namespace RTC
 
 			if (requested)
 			{
-				auto* storageItem = this->buffer[currentSeq];
-				RTC::RtpPacket* packet{ nullptr };
+				auto* storageItem = this->storageItemBuffer.Get(currentSeq);
+				RTC::RtpPacket::SharedPtr packet{ nullptr };
 				uint32_t diffMs;
 
-				// Calculate the elapsed time between the max timestampt seen and the
-				// requested packet's timestampt (in ms).
+				// Calculate the elapsed time between the max timestamp seen and the
+				// requested packet's timestamp (in ms).
 				if (storageItem)
 				{
-					packet = storageItem->packet;
+					packet = storageItem->originalPacket->Clone();
```

Comment:
If `storageItem` has a `clonnedPacket` we should not create it again but reuse it, since we are in the scenario where the packet was already resent and we did it before.

Reply:
Anyway, my previous comment [here](https://github.com/versatica/mediasoup/pull/675/files#r810548053) obsoletes this comment.